src/MigrationLog/MigrationLogModel.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Give\MigrationLog;
+
+/**
+ * Class MigrationLogModel
+ * @package Give\MigrationLog
+ *
+ * @since 2.9.7
+ */
+class MigrationLogModel {
```

Comment:
We discuss this and decided to log only succeded migrations. For everything else ( errors ), we will use the logs table. But, I see the benefit of having that column. This can be used to prevent failed migrations from trying to re-run all the time. Let's discuss this one more time.

Reply:
Resolved 87a3c8ce